### DISCUSSION TITLE
EXPS in Photo Mode doesn't display correct ISO during preview

### DISCUSSION THREAD

**Discussion by Squinkz** [gopro/labs#867] [General]
This isn't a big problem, as the photo is still captured with the correct ISO, but during preview on the camera's screen the ISO is displayed as being a stop or two higher than it should be.

Video demonstrating the issue: https://www.youtube.com/watch?v=VDaoD3N50gk

I took the camera into a very dark place so that the camera would try and use the highest ISO it could. I set ISO to maximum 100. EXPS displays it on screen as 375, and the preview reflects this level of gain, however when I snap the shutter it takes the picture at 100 ISO.  At 800 ISO max, preview is set to a whopping 3012! All other ISO settings preview higher than the max ISO setting, but the max ISO is still respected during capture. So yea, not a huge issue as it doesn't seem to affect performance, only preview.

This is only an issue in Photo Mode, Video Mode seems to be unaffected.

If interested, these are the labs settings I have activated:

WIDE | 1
LOGB | 400
OWNR | [redacted]
EXPT | 2
GUID | 1.3334, 1.7778, 0.3809, 5.3334
GUIC | 255, 200, 100, 100
EXPS | 1
LEVL | 6
HIST | 1
HSTP | 5, 24, 50
AUDS | 1
EVBS | -0.5000
GCMP | 1
PRXY | 1
FAST | 1
DAMP | 3
TUSB | 1
USRD | 1

Oh yea, and I'm running latest Labs version, H23.01.02.20.70.

**Comment by dnewman-gpsw** [comment]
This is by design and so not a bug.  Although I agree for a photographer it might seem misleading.  This is the design reason I believe: during preview camera is actually in video mode, collecting as many frames as possible so you have a smooth image for photo framing, without having a big shutter lag of a long exposure.  To compensate for the shorter shutter of video, a higher ISO is used than what you may have set in your menu.  Most users would prefer smooth preview.

The only reason this is being noticed now (it has always been the case), is EXPS=1 was expanded to support non-video modes for improved solar eclipse setups.

**Reply by Squinkz** [reply]
Alright that actually makes a lot of sense regarding the shutter speed running in video mode. Thanks!
